Xamarin.Forms.Xaml/XmlnsHelper.cs

```diff
@@ -46,5 +59,17 @@ public static void ParseXmlns(string xmlns, out string typeName, out string ns,
 					typeName = decl;
 			}
 		}
+
+		static void ParseUsing(string xmlns, out string typeName, out string ns, out string asm, out string targetPlatform)
+		{
+			typeName = ns = asm = targetPlatform = null;
+
+			foreach (var decl in xmlns.Split(';')) {
+				if (decl.StartsWith("using:", StringComparison.Ordinal)) {
+					ns = decl.Substring(6, decl.Length - 6);
+					continue;
```

Comment:
fair enough. the `continue` doesn't hurt anything, just seemed superfluous at this time. :)